central/policy/service/service_impl.go

```diff
@@ -1042,7 +1046,7 @@ func flattenPolicyGroupMap(policyGroupMap map[string][]*storage.PolicyGroup) []*
 
 		var policyValueLists []*storage.PolicyValue
 		for _, policyGroup := range singleGroupList {
-			// For now we don't care which search term a policy value came from because no two search terms can
+			// For now we don't care which search term a policy value came f/rom because no two search terms can
```

Comment:
revert?

Reply:
I do not even know how I made this change. Thank you.